styles.css

```diff
@@ -9,612 +9,965 @@ body {
+  }
+
+  .desktop-img-0 {
+    /* border: 2px solid; */
+    background-image: url("images/Snapshoot-Portfolio2.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  .work-content {
+    display: flex;
+    align-self: center;
+    justify-content: center;
+    height: 448px;
+    margin-top: 20px;
+  }
+
+  .tonic0,
+  .tonic {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .prop {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .tonic-def {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  /* card 2  */
+
+  .work-container2 {
+    border: 1px solid #cbcdd1;
+    display: flex;
+    flex-direction: row-reverse;
+    width: 75%;
+    height: 580px;
+    background: #fff;
+    border-radius: 16px;
+    justify-content: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+  .tonic {
+    height: auto;
+    width: auto;
+  }
+
+  .desktop-img-1 {
+    background-image: url("images/Snapshoot-Portfolio-4.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  .ruby {
+    border: 2px;
+    display: flex;
+    margin-left: 4px;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 12px;
+    width: 114px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    font-size: 14px;
+  }
+
+  /* card 3  */
+  .work-container3 {
+    border: 1px solid #cbcdd1;
+    display: flex;
+    width: 75%;
+    height: 580px;
+    background: #fff;
+    border-radius: 16px;
+    justify-content: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+  .desktop-img-2 {
+    background-image: url("images/Snapshoot-Portfolio.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  /* card 4  */
+  .work-container4 {
+    border: 1px solid #cbcdd1;
+    display: flex;
+    flex-direction: row-reverse;
+    width: 75%;
+    height: 580px;
+    background: #fff;
+    border-radius: 16px;
+    justify-content: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+  .desktop-img-3 {
+    background-image: url("images/Snapshoot-Portfolio-2.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  /* about  */
+  .introd {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
Kindly update the styling of your about section to better match the figma design🙏.

Your design:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/37116322/220734826-61fb50aa-1b3b-4b9d-98b0-a1dfb6b566e6.png">

Figma design:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/37116322/220734894-aebad330-bf70-4957-b6cd-9bd3ac6adf2a.png">

A few things to note:
- Please reduce the border radius at the top right
- Kindly increase the font weight of the `about me` text
- Kindly increase the width of the paragraph
- Kindly reduce the width of the languages section as its causing the text section to be squished
- Kindly update the styling of the technologies (HTML, CSS, JS). They should be column aligned
- Please slightly reduce the padding of the `get my resume` button